Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java

```diff
@@ -233,31 +233,8 @@ public int getNumberOfProcesses(String tasks) {
      */
     public String convertTasksToWorkflow(String tasks) {
         currentTasks = tasks;
-
-        try {
-            if (workflowAlreadyExist()) {
-                PrimeFaces.current().executeScript("PF('confirmWorkflowPopup').show();");
-                return this.stayOnCurrentPage;
-            }
-        } catch (DAOException e) {
-            Helper.setErrorMessage(ERROR_READING, new Object[] {ObjectType.TEMPLATE.getTranslationSingular() }, logger,
-                e);
-            return this.stayOnCurrentPage;
-        }
-        return createNewWorkflow();
-    }
-
-    private boolean workflowAlreadyExist() throws DAOException {
-        List<Task> processTasks = aggregatedProcesses.get(currentTasks).get(0).getTasks();
-        List<Template> allTemplates = ServiceManager.getTemplateService().getAll();
-        for (Template template : allTemplates) {
-            if (migrationService.tasksAreEqual(template.getTasks(), processTasks)) {
-                workflowToUse = template.getWorkflow();
-                return true;
-            }
-        }
-        return false;
-
+        PrimeFaces.current().executeScript("PF('confirmWorkflowPopup').show();");
+        return this.stayOnCurrentPage;
```

Comment:
Since you always return `null` here, I would suggest to change to function to return type `void` and remove the return statement entirely.